### DISCUSSION TITLE
Working CityEnviron For Linux

### DISCUSSION THREAD

**Discussion by miaris98** [microsoft/AirSim#4632] [Support Q&A] [answered]
I've seen similar issues on the topic but still there is not a clear way to get CityEnviron on Linux for newer versions any tips?

**Comment by jonyMarino** [comment, marked as answer]
Hi @miaris98! The current City project that we have doesn't compile for Linux. To have something similar you should find an environment like that one out there.

**Reply by miaris98** [reply]
@jonyMarino may i ask why the support is droped on newer versions? i tried the Env on windows and seems pretty good

**Reply by jonyMarino** [reply]
@miaris98 I'm not sure because since I work for AirSim, it was already broken. It might be that that project has a plugin inside that is not compatible with Linux
